### PR TITLE
docs: More helplink documentation

### PR DESCRIPTION
**Pull request by ynsehoornenborg**
Small batch to prevent major rework in case of general findings.

### CODE REVIEW

File: Documentation/Diagnostics/PH2125.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AlignOperatorCountAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AlignOperatorsCountAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+
+## Introduction
+
+Users expect some operators to exists on the same type. For this diagnostic the plus (`+`) operator or the minus (`-`) operator is declared, a user will expect the equality operators (`==` and `!=`) to be available also. For some operators the pairing rule is mandated by [microsoft](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/operator-overloading#overloadable-operators). This diagnostics extends the pairing to the plus, minus and equalality operators.
+
+This diagnostic implements rule [7@531](https://csviewer.tiobe.com/#/ruleset/rule?setid=4T_Jr6-VSX6fp6egDIhGow&status=CHECKED,UNCHECKED&tagid=6u27MkXORKaev0VNuWR_SA&ruleid=TCjgzMnETo2yCNKdNzCS0Q) of the Philips C# [Coding Standard](https://csviewer.tiobe.com/#/ruleset/intro?tagid=6u27MkXORKaev0VNuWR_SA&setid=4T_Jr6-VSX6fp6egDIhGow).
```
Comment (bcollamore) [inline Documentation/Diagnostics/PH2125.md:16]: The other similar rules don't mention the standard?

**Review by bcollamore — APPROVED** [review]

File: Documentation/Diagnostics/PH2125.md (hunk 1 above)
Comment (ynsehoornenborg) [reply Documentation/Diagnostics/PH2125.md:16]: Correct, the others are not in the Philips Coding Standard

File: Documentation/Diagnostics/PH2098.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+{
+    public void GoodMethod()
+    {
+        try
+        {
+            return 0;
+        }
+        catch (Exception ex) 
+        {
+            Log(ex);
```
[the remark this answers is not in the document]
Comment (ynsehoornenborg) [reply Documentation/Diagnostics/PH2098.md:51]: Done

File: Documentation/Diagnostics/PH2096.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2096 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidAsyncVoidAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidAsyncVoidAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+
+## Introduction
+
+[MSDN magazine](https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void) can explain this better than we could.
```
[the remark this answers is not in the document]
Comment (ynsehoornenborg) [reply Documentation/Diagnostics/PH2096.md:14]: Corrected

File: Documentation/Diagnostics/PH2089.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2089 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidAssignmentInConditionAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidAssignmentInConditionAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+
+## Introduction
+
+Avoid placing variable assignment inside of the condition part of an `if` or `?` statement. The notion of resulting value from suchg a statement is not widely known in the C# community. This makes the `if` statement hard to read and debug.
```
[the remark this answers is not in the document]
Comment (ynsehoornenborg) [reply Documentation/Diagnostics/PH2089.md:14]: Corrected

**Review by bcollamore — APPROVED** [review]

**Pull request merged**